app/controllers/listings_controller.rb

```diff
@@ -2,8 +2,8 @@ class ListingsController < ApplicationController
   before_action :set_listing, only: %i[ show edit update destroy ]
 
   def index
-      @listings = Listing.all
-    end
+    @listing = Listing.all
+    @user = User.find_by(email: current_user.email)
```

Comment:
Before we commit this - can you clarify why we need the @user line for index? (not saying it's wrong - just want to understand it for the code)
I think we need it in new/edit for sure

Reply:
I didn't have a user line in the controller or did you add that?

Reply:
I think I added it to new but it went into index by mistake! My bad